client/my-sites/upgrades/checkout-thank-you/custom-domain-purchase-detail.jsx

```diff
@@ -12,22 +12,24 @@ import { hasCustomDomain } from 'lib/site/utils';
 
 const CustomDomainPurchaseDetail = ( { selectedSite, hasDomainCredit, translate } ) => {
 	const renderClaimCustomDomain = () =>
-		<PurchaseDetail
-			icon="globe"
-			title={ translate( 'Select Your Custom Domain' ) }
-			description={
-				translate(
-					'Your plan includes a free custom domain. Replace {{em}}%(siteDomain)s{{/em}} ' +
-					'with a custom domain to personalize your site.',
-					{
-						args: { siteDomain: selectedSite.domain },
-						components: { em: <em /> }
-					}
-				)
-			}
-			buttonText={ translate( 'Claim your free domain' ) }
-			href={ `/domains/add/${ selectedSite.slug }` }
-		/>;
+		selectedSite.plan.user_is_owner
```

Comment:
> So, instead of grabbing user_is_owner out of selectedSite.plan, let's use a selector for this in the outside connected component and pass it down to the CustomDomainPurchaseDetail component.

@lamosty  Seems reasonable, but let's not block the PR. How about we follow up in another Janitorial?